src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -154,10 +154,19 @@ public boolean hasConflictingInterview(Interview interview) {
         return interviewSchedule.hasConflictingInterview(interview);
     }
 
-    /*@Override
-    public void deleteInterview(Interview target) {
-        interviewSchedule.removeInterview(target);
-    }*/
+    @Override
+    public void deleteInterview(Candidate target) {
```

Comment:
Hi En en! Possible to rename this method to something like `deleteInterviewForCandidate` or something along that line? Because 1) the method sounds abit confusing now and 2) this method will be used for actual deletion of interviews!

Reply:
Sure, thought I could rip off from the `addInterview` method :P
Have made the necessary changes! 🥂